code/index.html

```diff
@@ -1,30 +1,152 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Go Green</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <!-- Hero Header -->
+  <header id="hero" class="hero-wrapper-container hero-styling">
+    <div class="hero-inner-container hero-video">
+      <video class="video" id="video" loop muted autoplay playsinline>
+        <source src="forest.mp4" type="video/mp4">
+      </video>
+    </div>
+    <div class="title title-container">
+      <h1>Lets shape the future together!</h1>
+      <p>Become a volunteer today</p>
+      <button class="btn">Learn more</button>
+    </div>
+  </header>
+
+  <!-- Signup form -->
+  <div class="info-section-wrapper">
+    <form id="form" class="wrapper-container" action="http://httpbin.org/anything" method="POST" autocomplete="on">
+
+      <!-- first three inputs -->
+      <section class="inputs-wrapper">
+        <label class="child-form-container" for="name">
+          <input class="input custom-input" id="name" type="text" name="name" placeholder="Name" value=""
+            pattern="[a-zA-Z ]{1-32}" oninvalid="setCustomValidity('Please enter alphabets only.')" required>
+          <div class="tip" id="tip">&#128161 Example: Jane Ostin.</div>
+        </label>
+
+        <label class="child-form-container" for="email">
+          <input class="input custom-input" id="email" type="email" name="email" placeholder="Email" value=""
+            oninvalid="setCustomValidity('Please enter valid email.')" required>
+          <div class="tip" id="tip">&#128161 Example: your_email@gmail.com</div>
+        </label>
+
+        <label class="child-form-container" for="phone">
+          <input class="input custom-input" id="phone" type="tel" minlength="10" maxlength="14" name="phone"
+            pattern="^(?:0|\(?\+33\)?\s?|0033\s?)[1-79](?:[\.\-\s]?\d\d{4}$"
+            oninvalid="setCustomValidity('Please enter valid mobile number.')" placeholder="Phone Number" value=""
+            required>
+          <div class="tip" id="tip">&#128161 Should contain at least 10 digit.</div>
+        </label>
+      </section>
+
+      <!-- radio buttons set -->
```

Comment:
Great work implementing radio buttons! 
And also now I learned how one of the radio buttons is "checked" from the start. Lovely. Beautifully done.

Reply:
This i checked hack I first learned  from the technigo code along , they had provided a cool code along for radio buttons.